examples/informer/README.md

```diff
@@ -0,0 +1,41 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type and handle to create/update/delete events
+
+## Running
+
+``` go
+# assumes you have a working kubeconfig, not required if operating in-cluster 
+go run *.go -kubeconfig=$HOME/.kube/config
+```
+
+## Use Cases
+
+* Capturing resource events for logging to external systems (eg. monitor non-Normal Events and publish metrics to a time series database)
```

Comment:
I'm addressing some of the remaining code-review comments in a new branch against kubernetes/kubernetes, and fixed this in https://github.com/kubernetes/kubernetes/commit/9645bbf8071304a4e835eb323e6f2d67e89562d4